### PR TITLE
[docs] batch02 of updating functions

### PR DESCRIPTION
**Pull request by edgar2020**
### Description

Add examples to the sql-functions.md page

Updates the LOG10, POWER, SQRT, TRUNC, TRUNCATE, ROUND, and MOD numeric functions.

This PR has:

- [X] been self-reviewed.

### CODE REVIEW

File: docs/querying/sql-functions.md [hunk 1]
```diff
@@ -1317,11 +1351,25 @@ Returns the one-based index position of a substring within an expression, option
...
+Calculates a numerical expression raised to the specified power.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `POWER(base, exponent)`
+* **Function type:** Scalar, numeric
 
-Calculates a numerical expression raised to the specified power.
+<details><summary>Example</summary>
+
+The following example raise 5 to the power of 2.
```
Comment (asdf2014) [inline docs/querying/sql-functions.md:1361]:
```suggestion
The following example raise 5 to the power of 2.

```

File: docs/querying/sql-functions.md [hunk 2]
```diff
@@ -1133,11 +1152,26 @@ Returns the minimum value of a set of values.
...
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `MOD(x, y)`
+* **Function type:** Scalar, numeric
+
+<details><summary>Example</summary>
+
+The following calculates 78 MOD 10.
+
+```sql
+SELECT MOD(78, 10) as "mod"
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md:1165]: nit: suggest using an english word for the alias (kind of the point of an alias :D )

File: docs/querying/sql-functions.md [hunk 3]
```diff
@@ -1446,11 +1515,26 @@ Calculates the trigonometric sine of an angle expressed in radians.
...
+Calculates the square root of a numeric expression.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `SQRT(<NUMERIC>)`
+* **Function type:** Scalar, numeric
 
-Calculates the square root of a numeric expression.
+<details><summary>Example</summary>
+
+The following example applies SQRT to the `trip_distance` column from the `taxi-trips` datasource.
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md:1525]: This appears to be a static select that does not use the data from `taxi-trips`

**Review by techdocsmith — CHANGES_REQUESTED** [review]
Request minor changes. Otherwise looking good.

**Review by edgar2020 — COMMENTED** [review]
Comments have been addressed

File: docs/querying/sql-functions.md [hunk 4]
```diff
@@ -1398,11 +1447,31 @@ Returns the rightmost number of characters from an expression.
...
+Calculates the rounded value for a numerical expression.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `ROUND(expr[, digits])`
+* **Function type:** Scalar, numeric
 
-Calculates the rounded value for a numerical expression.
+<details><summary>Example</summary>
+
+The following applies the ROUND function to 0 decimal points on the `pickup_longitude` column. The `pickup_longitude` column is from the `taxi-trips` datasource.
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md:1457]:
```suggestion
The following applies the ROUND function to 0 decimal points on the `pickup_longitude` column from the `taxi-trips` datasource.
```

File: docs/querying/sql-functions.md [hunk 5]
```diff
@@ -1720,20 +1804,41 @@ Trims the leading or trailing characters of an expression.
...
+Truncates a numerical expression to a specific number of decimal digits.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `TRUNCATE(expr[, digits])`
+* **Function type:** Scalar, numeric
 
-Truncates a numerical expression to a specific number of decimal digits.
+<details><summary>Example</summary>
 
+The following applies the TRUNCATE function to 1 decimal place on the `pickup_longitude` column. The `pickup_longitude` column is from the `taxi-trips` datasource.
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md:1823]:
```suggestion
The following applies the TRUNCATE function to 1 decimal place on the `pickup_longitude` column from the `taxi-trips` datasource.
```

File: docs/querying/sql-functions.md [hunk 6]
```diff
@@ -1398,11 +1447,31 @@ Returns the rightmost number of characters from an expression.
...
 
-Calculates the rounded value for a numerical expression.
+<details><summary>Example</summary>
+
+The following applies the ROUND function to 0 decimal points on the `pickup_longitude` column. The `pickup_longitude` column is from the `taxi-trips` datasource.
+
+```sql
+SELECT
+  "pickup_longitude" AS "pickup_longitude",
+  ROUND("pickup_longitude", 0) as "round_pickup_longitude"
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md:1462]:
```suggestion
  ROUND("pickup_longitude", 0) as "rounded_pickup_longitude"
```
nit

File: docs/querying/sql-functions.md [hunk 7]
```diff
@@ -1398,11 +1447,31 @@ Returns the rightmost number of characters from an expression.
...
+FROM "taxi-trips"
+WHERE "pickup_longitude" IS NOT NULL
+LIMIT 1
+```
+Returns the following:
+
+| `pickup_longitude` | `round_pickup_longitude` |
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md:1469]:
```suggestion
| `pickup_longitude` | `rounded_pickup_longitude` | 
```

**Review by techdocsmith — APPROVED** [review]
LGTM after minor suggestions

**Pull request merged**